core/templates/dev/head/css/oppia.css

```diff
@@ -1549,6 +1790,12 @@ md-card.oppia-dashboard-tile {
   line-height: 1.846;
 }
 
+.toast {
```

Comment:
Here and elsewhere, why are we introducing so many new general classes? I would expect to see a corresponding deletion if this is something that we did before.

Reply:
Hi @seanlip, this is used to override the CSS of bootstrap-4